src/components/OnboardingComponents/SemanticSearch/QuerySuggestions.tsx

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+import {Box, Chip, Typography} from '@mui/material';
+import {useTranslation} from 'react-i18next';
+
+interface QuerySuggestionsProps {
+  /** A list of suggestion strings to display. */
+  suggestions: string[];
+  /** The function to call when a suggestion chip is clicked. */
+  onSuggestionClick: (query: string) => void;
+}
+
+/**
+ * A component that displays a list of clickable query suggestion chips.
+ */
+export default function QuerySuggestions({suggestions, onSuggestionClick}: QuerySuggestionsProps): JSX.Element {
+  const {t} = useTranslation('global');
+
+  return (
+    <Box mb={2}>
+      <Typography variant='caption' sx={{color: 'GrayText', mb: 1, display: 'block'}}>
+        {t('semanticSearch.suggestions.title')}
+      </Typography>
+      <Box display='flex' flexWrap='wrap' gap={1}>
+        {suggestions.map((suggestion) => (
+          <Chip
+            key={suggestion}
+            label={suggestion}
+            onClick={() => onSuggestionClick(suggestion)}
+            variant='outlined'
+            size='small'
+            sx={{
+              cursor: 'pointer',
+              borderColor: 'primary.light',
+              color: 'primary.main',
+              '&:hover': {
+                backgroundColor: 'primary.main_10', // Assuming a 10% opacity color from your theme
```

Comment:
not sure if material does this behind the scenes. It seems to work though 🤔
the Theme is created in `src\util\Theme.ts` jfyi.
```suggestion
                // Applying the primary color at 10% opacity from the theme.
                backgroundColor: 'primary.main_10',
```